### PR TITLE
Venueslist

### PR DESCRIPTION
**Pull request by jojo12**
A simple list of venues together with cities. Was proposed long time ago from @Bluefox in the forums.

### CODE REVIEW

File: site/classes/output.class.php [hunk 1]
```diff
@@ -990,6 +990,65 @@ static function flyer($data, $image, $type, $id = null)
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Hoffi1) [inline site/classes/output.class.php]: What's that?

Comment (jojo12) [inline site/classes/output.class.php]: strange, I did not made this: Now are only the changed raws in! (is this made automaticly?

**Comment by Hoffi1** [comment]
I want to release 2.2.1 first.
Then it could become part of 2.3.0 but at the moment I've no time for review. Without looked into the files I generally don't like to add more models/views - we already have a lot of them, often nearly identical in wide parts of code, but eating a lot of time when changes must be made on all of them one by one. Most of them were born by duplicating another one and changing a bit here and there.
It would be much smarter and easier to maintenance if already existing venues model will be used with more configurable venues view or maybe dedicated venues list view.

**Comment by jojo12** [comment]
It's ok you're the one who decides. 
Of course the importance of the different models depend on the view of the users. 
In my sight, the venues_view is very few used, because with a lot of venues, it's not handy. So the simple venueslist helps find the one venue and leads to this one venue view. This is very good and a missing link.

**Comment by depebo** [comment]
@jojo12 do we still need this to go into next JEM release?

**Comment by jojo12** [comment]
yes, please: For lot of people it could be very useful (more useful than the venues part which is much too big.)
I used it for searching meetingplaces for about 20000 peoples, which was very helpful. See my exemple;
ortsliste

**Pull request merged**